lib/crowbar/client/command/batch/export.rb

```diff
@@ -27,6 +34,40 @@ def request
 
           def execute
             request.process do |request|
+              case request.code
+              when 200
+                if write(request.body)
+                  say "Successfully exported batch"
+                else
+                  err "Failed to export batch"
+                end
+              else
+                err request.parsed_response["error"]
+              end
+            end
+          end
+
+          protected
+
+          def write(body)
+            path.binmode
+            path.write body
+
+            true
+          rescue
+            false
+          end
+
+          def path
+            @path ||= case args.file
+            when "-"
```

Comment:
Style/CaseIndentation: Indent `when` as deep as `case`. (https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)